js/data/currencies.js

```diff
@@ -651,6 +651,10 @@ function getIndexedCurrencies() {
   return _indexedCurrencies;
 }
 
+export function getListofCurrencies() {
+  return _.pluck(currencies, 'code');
+}
```

Comment:
I think I'd need more information on the error you're getting in the test, since you shouldn't have an undefined currency list since it's being declared in the module.

There may be some other issue and I'd rather fix the source rather than mask it with a work-around.

I'm not opposed to bringing in a getListofCurrencies function, but I don't really see a use case for it given that getTranslatedCurrencies() and getCurrencyByCode() should be able to satisfy most things.

If it were to stay, I'd want to memoize it so the list isn't being traversed on each call... but I'm doubting its need.